app/controllers/main/routes.py

```diff
@@ -26,6 +27,14 @@ def volunteerIndicateInterest():
                            interests_ids = interests_ids)
 
 
+@main_bp.route('/rsvpForEvent', methods = ['POST'])
+def RSVPCaller():
+    print("Hello")
+    eventData = request.form.copy()
```

Comment:
is there a reason to copy the event data first?

Reply:
nope. in fact this function is unnecessary